documents/implementation-guide.md

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
can we also add the badges?

Reply:
The whole changes have not been commited !! 
Please take a look at the PR again ... Thank you